### PR TITLE
Enable saving metric type cards

### PR DESCRIPTION
**Pull request by metamben**
Part of #37348.

### CODE REVIEW

File: frontend/src/metabase/entities/questions.js [hunk 1]
```diff
@@ -40,55 +40,21 @@ const Questions = createEntity({
...
           get: async payload => {
             const get = GET("/api/card/:id");
-            const result = await get(payload);
-
-            if (result.name.toLowerCase().includes("metric")) {
-              return { ...result, type: "metric" };
-            }
-
-            return result;
+            return await get(payload);
```
Comment (kamilmielnik) [inline frontend/src/metabase/entities/questions.js:43]: This whole section should be removed, lines 31-95.

Comment (kamilmielnik) [reply frontend/src/metabase/entities/questions.js]: Done: f7651d414f8777c0ece2985c5b60ecfdf6331b76

**Review by ranquild — APPROVED** [review]

File: test/metabase/api/card_test.clj [hunk 2]
```diff
@@ -1035,6 +1035,14 @@
                :type    "question"}
               (mt/user-http-request :crowberto :post 200 "card" (card-with-name-and-query (mt/random-name))))))))
 
+(deftest create-card-with-metric-type
+  (mt/with-model-cleanup [:model/Card]
+    (testing "can create a metric card"
+      (is (=? {:dataset false
+               :type    "metric"}
+              (mt/user-http-request :crowberto :post 200 "card" (assoc (card-with-name-and-query (mt/random-name))
```
Comment (snoe) [inline test/metabase/api/card_test.clj:1043]: should we be checking that a metric card's query conforms to the rules around metrics (must have 1 aggregation)?

**Review by snoe — APPROVED** [review]
LGTM - I assume the query check can be addressed in the future.

**Pull request merged**